vrk-kpa/suomidigi
# Suopa 596 group poc

TO TEST:
- make fresh
- If you get a nag for rebuild permissions, rebuild them here: https://suomidigi.docker.sh/admin/reports/status/rebuild
- Go here: https://suomidigi.docker.sh/admin/group
- Create a new group with admin
- Add some paragraphs to the group to make it look like the front page of the group.
- Create content from the group controls (left on the group's page). 
  - One page/article that is published
  - One page/article that is unpublished
- Use incognito tab and check for the content as an anonymous:
  - Group's front page should be visible
  - Published content should be visible
  - Unpublished content should NOT be visible.
- As an admin, add your standard user to the group as content producer. 
- In incognito, log in as yourself and check that you can see even the unpublished content (and edit it).
